src/Modal/elements.js

```diff
@@ -49,9 +49,9 @@ export const Dialog = styled.dialog`
   background: ${({ theme: { palette } }) => palette.white};
 
   /* For small devices */
-  ${breakpoint('xs', 'sm')`
-    width:90%;
-    margin-left: calc(-90% / 2)};
+  ${breakpoint('xs', 'md')`
+    width: 50rem;
```

Comment:
Is that a good thing to have a `rem` value for the modal on little device ? I thought we've talking about the full size modal with `100%` width on small devices, @spartDev

Reply:
@sun-tea @AntoninMarchardDev I think we should speak with @arthurmonnet about that!

Reply:
I think we could ask @arthurmonnet what is the default behavior of the modal on the small device

Reply:
100% width and full screen with fixed header & fixed bottom. I added an example on Zeplin --> https://app.zeplin.io/project/5c08e7dab171cd2fd787d7d6/screen/5c3328597c1f1c4f0c57b52e

Reply:
thx